src/components/Selection/Selection.tsx

```diff
@@ -0,0 +1,305 @@
+            })
+        : defaultIcon(kind, responsiveMode);
+
+    return (
+        <div
+            {...omitProps(passThroughs, undefined, _.keys(Selection.propTypes))}
+            className={cx(
+                '&',
+                `&-is-${responsiveMode}`,
+                kind && `&-${kind}`,
+                {
+                    '&-has-background': hasBackground,
+                    '&-is-bold': isBold,
+                    '&-is-filled': isFilled,
+                    '&-is-top': isTop,
+                    '&-no-title': _.isEmpty(labelProps),
+                },
+                className
+            )}
+        >
+            {icon}
+
+            <div className={cx('&-content')}>
+                <div className={cx('&-label-container')}>
+                    <span
+                        {...labelProps}
+                        className={cx('&-label', isSmall && '&-label-is-small')}
+                    />
+
+                    {isRemovable ? (
+                        <CloseIcon
+                            isClickable
+                            size={!isSmall ? 8 : 16}
+                            className={cx(
+                                '&-close-button',
+                                isSmall && '&-close-button-is-small'
+                            )}
+                            onClick={({ event }) => { onRemove({ event, props }) }}
+                        />
+                    ) : null}
+                </div>
+                {!_.isEmpty(children) &&
+                    <div className={cx('&-children-container')}>
+                        {_.map(React.Children.toArray(children), (child, i) => {
+                            if (React.isValidElement(child) && child.type === Selection) {
+                                return (
+                                    <Selection
+                                        key={
+                                            _.get(
+                                                getFirst(child.props, Selection.Label),
+                                                ['props', 'children'],
+                                                {}
+                                            ) + i
+                                        }
+                                        {...child.props}
+                                    />
+                                );
+                            }
+                            return child;
+                        })}
+                    </div>
+                }
+            </div>
+        </div>
+    );
+}
+
+Selection.displayName = 'Selection';
+Selection.Icon = SelectionIcon;
+Selection.Label = SelectionLabel;
+Selection.peek = {
+            description: `
+                Used to indicate selections. It's very similar to \`Tag\` but is meant
+                to be used in areas of the UI that have more space available to them.
+            `,
+            categories: ['communication'],
+        }
```

Comment:
More of a UX issue: The way this text shows up in Storybook "Notes" could be clearer. Because Selection is off the the left so far.  "It's" lacks a clear referent, so a fix could be something like: `Used to indicate selections. Selection is very similar to \`Tag\` but` ...
<img width="1613" alt="Screen Shot 2019-11-20 at 5 45 00 PM" src="https://user-images.githubusercontent.com/19521671/69293413-a1d47000-0bbd-11ea-8394-c5ae5c307806.png">

Reply:
Good point, I agree that is more clear. I'll change this to say `Selection is...`